broker-cli/utils/error-handler.js

```diff
@@ -1,13 +1,11 @@
-
 /**
  * Takes in an error object and throws a friendly error if the broker daemon is down
  *
  * @param {Error} error
  * @return {void}
  */
-
 function handleError (error) {
-  if (error.message === '14 UNAVAILABLE: Connect Failed') {
+  if (error.message && error.message === '14 UNAVAILABLE: Connect Failed') {
```

Comment:
seems like an unnecessary addition. Did you mean to do `if (error && error.message ==...)`?

Reply:
Nope, this was just a brain fart. thanks for the catch